php/libraries/Candidate.class.inc

```diff
@@ -504,12 +507,21 @@ class Candidate
     /**
      * Return the candidate's sex
      *
-     * @return string|null Male|Female|Other String describing the sex
-     *                     and null if scanner
+     * @return ?Sex
+     */
+    function getCandidateSex(): ?Sex
+    {
+        return new Sex($this->candidateInfo["Sex"]);
```

Comment:
I don't see why this should return a new one every time, and not create one in the constructor and return the same one.